react/src/components/HitsList.jsx

```diff
@@ -1,7 +1,11 @@
 import React from 'react';
 
-const HitsList = (props) => {
-    return null;
-};
+const HitsList = ({items}) => {
+    return (
+        <ul className="hits">
+            {items?.length > 0 ? (items.map(item => item)) : <h2 className="error">No results found!</h2>}
```

Comment:
Faptul ca operatia de cautare nu intoarce niciun rezultat nu ar trebui sa insemne neaparat ca s-a produs o eroare.
Poate ar fi mai potrivit sa schimbi numele clasei din _error_ in _no-results_.